docs/user-event/api-convenience.mdx

```diff
@@ -55,6 +97,36 @@ hover(element: Element): Promise<void>
 pointer({target: element})
 ```
 
+```js
+const TestComponent = () => {
+  const [isHover, setIsHover] = useState(false)
+
+  return (
+    <div
+      style={{backgroundColor: isHover ? 'red' : 'green'}}
+      onMouseEnter={() => setIsHover(true)}
+      onMouseLeave={() => setIsHover(false)}
+    >
+      Hover
+    </div>
+  )
+}
+
+test('hover/unhover', async () => {
+  render(<TestComponent />)
+
+  const hoverBox = screen.getByText('Hover')
+
+  await userEvent.hover(hoverBox)
+
+  expect(hoverBox).toHaveStyle({backgroundColor: 'red'})
```

Comment:
Could you update this expectation please?
For example, to update the text of the `div` element instead.
The reason for this is that I personally don't like the `toHaveStyle` expectations, and IIRC some of the testing environments don't support this.

Reply:
@timdeschryver 
Oh, thank you.
I have modified the example code according to your comments.